great_expectations/dataset/pandas_dataset.py

```diff
@@ -318,6 +319,8 @@ def inner_wrapper(
             else:
                 raise ValueError(f"Unknown value of ignore_row_if: {ignore_row_if}")
 
+            validate_mostly(mostly)
```

Comment:
Support for computing `mostly` in V2's `MulticolumnMapExpectation` existed but validation did not so I added it.